frontend/ui/widget/textboxwidget.lua

```diff
@@ -860,6 +860,13 @@ function TextBoxWidget:_renderText(start_row_idx, end_row_idx)
     self:_renderImage(start_row_idx)
 end
 
+-- Lay out the full text, starting at the current line.
+-- (NOTE: This instantiates the inner bb (self._bb), so be careful about its lifecycle when you call this,
+--        c.f., TextBoxWidget:update).
+function TextBoxWidget:_updateLayout()
```

Comment:
Possibly `_updateTextLayout`, as it's not exactly the complete layout, @poire-z?

(Although, given the widget's name, text *is* most of the layout involved ;p).

Reply:
Either names are fine by me, as you prefer :)
(Althought the shorter _updateLayout() is just fine :)